lib/sync/installation.js

```diff
@@ -107,9 +107,9 @@ module.exports.processInstallation = (app, queues) => {
 
     const handleGitHubError = (err) => {
       const ignoredErrorTypes = ['MAX_NODE_LIMIT_EXCEEDED']
-      const notIgnoredError = err.errors.filter(error => !ignoredErrorTypes.includes(error.type)).length
+      const isIgnoredError = err.errors && err.errors.filter(error => ignoredErrorTypes.includes(error.type)).length
```

Comment:
I like that we're asserting positives. Easier on my brain. 😄 To your earlier point, does this change cause us to swallow not ignored errors when they are accomanied by an ignored one?

For example, if `err.errors == [{type: "MAX_NODE_LIMIT_EXCEEDED"}, {type: "Some other error"}]`.

Reply:
Good point! I had originally updated this in an attempt to fix the CI failures, just reverted this change for this PR so we don't change the functionality here.